services/lamda/vars.tf

```diff
@@ -0,0 +1,21 @@
+variable "AllowExecutionFromSNS" {
+    description = " policy has the permissions that the function needs to write logs to CloudWatch Logs."
+}
+
+variable "InvokeFunction" {
+    description = " this will allow to invoke aws lamda function"
+  
+}
+
+
+variable " function_name" {
```

Comment:
Two things:
- Space should not be used in a variable name
- change the function name to something better